### PR TITLE
feat: support input dirName in create-waku

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

**Review by Himself65 — COMMENTED** [review]
I dont know how to test this in real production

File: contrib/create-waku/package.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "name": "create-waku",
   "version": "0.4.1",
-  "bin": "./cli.js"
+  "bin": "./cli.js",
+  "files": [
+    "cli.js"
+  ],
+  "dependencies": {
+    "@clack/core": "=0.3.2",
```
Comment (dai-shi) [inline contrib/create-waku/package.json:9]: I've never seen this format.
What's the difference from `"0.3.2"`?
ref: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#dependencies

**Comment by dai-shi** [comment]
> I dont know how to test this in real production

Me neither.
(What I did was simply running cli.js...)

Besides that, as you see it's just a naive js for now. If we were to add dependencies, I would like to make it better:
- find the latest tag automatically
- get list of files automatically

**Comment by Himself65** [comment]
well, I will publish a `create-test-waku` package to try this in real world

File: contrib/create-waku/package.json (hunk 1 above)
Comment (Himself65) [reply contrib/create-waku/package.json:9]: This is my mistake. But I works correctly🤔️

**Comment by Himself65** [comment]
I want to wait until https://github.com/dai-shi/waku/pull/62 merged since it use monorepo correctly

**Comment by childrentime** [comment]
@himself65 https://github.com/vercel/next.js/blob/canary/test/integration/create-next-app/templates-pages.test.ts how about this

**Comment by himself65** [comment]
> @himself65 https://github.com/vercel/next.js/blob/canary/test/integration/create-next-app/templates-pages.test.ts how about this

nice example

**Comment by dai-shi** [comment]
superseded by #115

**Pull request closed without merging**